src/features/movieDetails/index.js

```diff
@@ -1,77 +1,100 @@
 import { useParams } from "react-router-dom";
 import { useEffect } from "react";
 import { useSelector, useDispatch } from "react-redux";
-import { getDetailsForMovie, selectDetails, selectCast, selectCrew } from '../movieSlice';
+import { getDetailsForMovie, selectDetails, selectCast, selectCrew, selectStatus } from '../movieSlice';
 import { MainWrapper } from "../../common/MainWrapper";
-import poster from "./backdrop.jpg";
 import { Backdrop, BackdropWrapper, PosterWrapper, Title, Wrapper } from "./styled";
 import { BackdropVotes } from "../../common/Tiles/Votes";
 import { MovieDescriptionTile } from "../../common/Tiles";
 import PersonTile from "../../common/Tiles/PersonTile";
+import Loader from "../../common/Loader";
+import { Error } from "../../common/Error";
+import { nanoid } from "@reduxjs/toolkit";
 
 const MovieDetails = () => {
+    const secureBaseUrl = "https://image.tmdb.org/t/p/original";
     const dispatch = useDispatch();
     const { id } = useParams();
 
     useEffect(() => {
         dispatch(getDetailsForMovie({ movieId: id }));
     }, [id, dispatch]);
+    const status = useSelector(selectStatus);
     const details = useSelector(selectDetails);
-    //const cast = useSelector(selectCast);
-    //const crew = useSelector(selectCrew);
+    const cast = useSelector(selectCast);
+    const crew = useSelector(selectCrew);
 
     return (
-        <p>{details.original_title}</p>
-
-        /* <MainWrapper
-            backdrop={
-                <BackdropWrapper>
-                    <Title>
-                        {details.original_title}
-                    </Title>
-                    <PosterWrapper>
-                        <Backdrop
-                            src={poster}
+        status === "loading" ?
+            <Loader /> :
+            status === "error" ?
+                <Error /> :
+                <MainWrapper
+                    backdrop={
+                        <BackdropWrapper>
+                            <Title>
+                                {details.original_title}
+                            </Title>
+                            <PosterWrapper>
+                                <Backdrop
+                                    src={
+                                        details.backdrop_path ?
+                                            `${secureBaseUrl}${details.backdrop_path}` :
+                                            ""//`${noPerson}`
+                                    }
+                                    alt="backdrop"
+                                />
+                            </PosterWrapper>
+                            <BackdropVotes
+                                voteAverage={details.vote_average}
+                                voteCount={details.vote_count}
+                            />
+                        </BackdropWrapper>
+                    }
```

Comment:
When we don't have a Backdrop Poster, we don't want to display the entire section, not just the poster

Reply:
aha, ok, You're right. It will be easy to change